src/ds5/ds5-options.cpp

```diff
@@ -37,6 +37,25 @@ namespace librealsense
                         "Emitter select, 0-disable all emitters, 1-enable laser, 2-enable laser auto (opt), 3-enable LED (opt)")
     {}
 
+    emitter_frequency::emitter_frequency(
+        uvc_sensor & ep, const std::map< float, std::string > & description_per_value )
+        : uvc_xu_option( ep,
+                         ds::depth_xu,
+                         ds::DS5_EMITTER_FREQUENCY,
+                         "Controls the emitter frequency, 57 [KHZ] / 91 [KHZ]",
+                         description_per_value )
+    {
+    }
+
+    void emitter_frequency::set( float value )
+    {
+        if( _ep.is_streaming() )
```

Comment:
Is this method needed?
For the emitter_option (see several lines above), since it inherits from uvc_xu_option - as this new option does - the set method has not been overriden, becasue it only calls uvc_xu_option::set, as done here.

Reply:
This override was made to enforce a requirement that no setting during streaming is allowed,
The parent  `uvc_xu_option::set` does not enforce that

Reply:
ok